tfjs-backend-webgpu/src/benchmark_ops_test.ts

```diff
@@ -88,7 +88,9 @@ describeWebGPU('Ops benchmarks', () => {
     expect().nothing();
   }
 
-  it('argMax', async () => {
+  // Failing on MacOS Timeout
+  // tslint:disable-next-line: ban
+  xit('argMax', async () => {
```

Comment:
For SwiftShader, yes, it is. For a real GPU device, it's not. But currently the bots only support swiftShader, so this case is skipped.

Reply:
In case this is a perf test and timeout simply means we can't achieve good perf on SwiftShader, I'm OK with this regression. 
Furthermore, I suggest to remove this file as a total, as we no longer track the perf here.

Reply:
Will remove `benchmark_ops_test.ts` in a separate PR. Thanks.